src/test/unit/controls/Line/Line-spec.js

```diff
@@ -70,9 +70,42 @@ describe("Line", () => {
         });
         it("throws error when no values are provided", () => {
             expect(() => {
-                graphDefault.loadContent(new Line(getInput([], false, false)));
+                graphDefault.loadContent(
+                    new Line(getInput(undefined, false, false))
+                );
             }).toThrowError(errors.THROW_MSG_NO_DATA_POINTS);
         });
+        it("display the legend when empty array is provided as input", () => {
```

Comment:
Can you add this to the legend section instead? It will be easy to move if we ever decide to split the unit tests into separate files. (The files are getting long and hard to maintain)

Somewhere around [here](https://github.com/cerner/carbon-graphs/blob/8e61d42a04879ed922537f04c3d68f447bc725c5/src/test/unit/controls/Line/Line-spec.js#L943)